nixos/doc/manual/release-notes/rl-2511.section.md

```diff
@@ -165,6 +165,21 @@
 
 - `services.gitea` supports sending notifications with sendmail again. To do this, activate the parameter `services.gitea.mailerUseSendmail` and configure SMTP server.
 
+- Revamp of the ACME certificate acquisication and renewal process to help scale systems with lots (100+) of certificates.
+
+  Units and targets have been reshaped to better support more specific dependency propagation and avoid
+  superfluously triggering unchanged units:
+
+  If a service requires a syntactically valid certificate to start it should now depend on the `acme-{certname}.service` unit.
+
+  We now always generate initial self-signed certificates as this drastically simplifies the dependency structure. As a result, the option `security.acme.preliminarySelfsigned` has been removed.
+
+  Instead of the previous `acme-finished-{certname}.target`s there are now `acme-order-renew-{certname}` services that will be activated
```

Comment:
```suggestion
  Instead of the previous `acme-finished-{certname}.target`s there are now `acme-order-renew-{certname}.service`'s that will be activated
```
That will probably make the change clearer.